'make' fails on Ubuntu 18.04 and 20.04

I am trying to run specification tests for v1.1.0 on Ubuntu 18.04 and 20.04, but they seem to fail due to building problems of Grammatica:
```
+ cd grammatica-1.6
+ ant -k
Buildfile: /home/andrius/optimade/specification/tests/tools/grammatica-1.6/build.xml

codegen:

compile-clean:
    [mkdir] Created dir: /home/andrius/optimade/specification/tests/tools/grammatica-1.6/classes

compile-java:
    [javac] Compiling 96 source files to /home/andrius/optimade/specification/tests/tools/grammatica-1.6/classes
    [javac] warning: [options] bootstrap class path not set in conjunction with -source 1.4
    [javac] error: Source option 1.4 is no longer supported. Use 6 or later.
    [javac] error: Target option 1.5 is no longer supported. Use 1.6 or later.
Target 'compile-java' failed with message 'Compile failed; see the compiler error output for details.'.
Target 'codegen' failed with message 'The following error occurred while executing this line:
/home/andrius/optimade/specification/tests/tools/grammatica-1.6/build.xml:76: Compile failed; see the compiler error output for details.'.
```
[snip]
```
BUILD FAILED
/home/andrius/optimade/specification/tests/tools/grammatica-1.6/build.xml:36: The following error occurred while executing this line:
/home/andrius/optimade/specification/tests/tools/grammatica-1.6/build.xml:76: Compile failed; see the compiler error output for details.
```
After that, all tests fail due to missing JAR. Pinging @sauliusg and @rartino.

By the way, starting with Ubuntu 20.04, Grammatica appears in standard Ubuntu repositories as `libgrammatica-java`.

--- a/dependencies/Debian-8.6/install.sh
+++ b/dependencies/Debian-8.6/install.sh
@@ -2,5 +2,6 @@
 
 sudo apt-get install -y \
     default-jdk \
+    patch \
     unzip \
 ;

--- a/tests/makefiles/Makelocal-tools
+++ b/tests/makefiles/Makelocal-tools
@@ -35,6 +35,7 @@ ${GRAMMATICA_LOG}:
 		|| (rm -vf ${GRAMMATICA_ZIP}; exit 12)
 	cd $(dir $@) && rm -rf $(notdir $(basename $@))
 	cd $(dir $@) && unzip ${GRAMMATICA_ZIP}
+	patch $(dir $@)/$(notdir $(basename $@))/build.xml $(dir $@)/grammatica-build.patch
 	cd $(dir $@); ( set -uex; cd $(notdir $(basename $@)); ant -k ) 2>&1 \
 		| tee $(notdir $@)
 

--- a/tests/tools/grammatica-build.patch
+++ b/tests/tools/grammatica-build.patch
@@ -1,0 +1,12 @@
+73,74c73,74
+<            source="1.4"
+<            target="1.5"
+---
+>            source="1.6"
+>            target="1.6"
+116,117c116,117
+<            source="1.4"
+<            target="1.5"
+---
+>            source="1.6"
+>            target="1.6"

